staging/prometheus-operator/patch/mesosphere/templates/rules/rules-1.14/kubernetes-system-apiserver.yaml

```diff
@@ -8,11 +8,11 @@ https://github.com/helm/charts/tree/master/stable/prometheus-operator/hack
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
-  name: {{ printf "%s-%s" (include "prometheus-operator.fullname" .) "kubernetes-system-apiserver" | trunc 63 | trimSuffix "-" }}
-  namespace: {{ template "prometheus-operator.namespace" . }}
+  name: {{ printf "%s-%s" (include "kube-prometheus-stack.fullname" .) "kubernetes-system-apiserver" | trunc 63 | trimSuffix "-" }}
```

Comment:
I need to take a look at this patch, this one was to fix the AggregatedAPIDown alerts that kept firing but I'll check if it has been fixed upstream and if not, modifying this file to be based off the latest upstream file and adding just the specific fix on top of it

Reply:
oh, hehe I made that change upstream https://github.com/prometheus-community/helm-charts/pull/455/files although forgot that this is an autogenerated file so at some point when a sync happens with wherever the upstream rules are generated.. this will be overwritten. haven't gotten around to understanding the syncing structure and what other upstream repos to look at to fix it there but we can just copy paste the same file here for now, even though it's the same at least in the future if it does get overwritten again this diff can remind us to reapply the fix (until it is fixed upstream of the upstream chart 🤔)

Reply:
^done